src/components/layout/Booking/BookingSuccess/BookingSuccess.jsx

```diff
@@ -32,6 +33,23 @@ const BookingSuccess = ({ quantity, price, token, isEvent, offerUrl }) => {
               </span>
             </Fragment>
           )}
+
+          {!isEvent && activationCode != null && (
+            <Fragment>
+              <span className="is-block">
+                {"Ton code d'activation : "}
+              </span>
+              <span className="is-block bs-activation-code">
+                {activationCode.code}
+              </span>
+
+              {activationCode.expirationDate != null && (
+                <span className="is-block">
+                  {` A activer avant le ${moment(activationCode.expirationDate).format('LL')}.`}
```

Comment:
```suggestion
                  {` à activer avant le ${moment(activationCode.expirationDate).format('LL')}.`}
```
Cette majuscule en milieu de phrase me paraît étonnante